README.md

```diff
@@ -1 +1,60 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+HTML and CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
I recommend you provide instructions on how to get a local copy of this project and install the required dependencies for it to run properly. It might not seem important for a project of this complexity, but it is better to get in the habit of providing proper `Getting Started` instructions for more complex projects later on.

Reply:
Done